src/Ejsmont/CircuitBreaker/Factory.php

```diff
@@ -69,4 +70,18 @@ public static function getMemcachedInstance(\Memcached $memcached, $maxFailures
         return new CircuitBreaker($storage, $maxFailures, $retryTimeout);
     }
 
+    /**
+     * Creates a circuit breaker with same settings for all services using redis instance as a backend
+     *
+     * @param \Redis $redis        instance of a connected Memcached object
+     * @param int   $maxFailures  how many times do we allow service to fail before considering it offline
+     * @param int   $retryTimeout how many seconds should we wait before attempting retry
+     *
+     * @return CircuitBreakerInterface
+     */
+    public static function getRedisInstance(\Redis $redis, $maxFailures = 20, $retryTimeout = 30) {

```

Comment:
I'm not sure about it, but maybe this force the user to install php5-redis to load this file, and so it would be wrong.